### PR TITLE
feat:View All Floating-Rates

### PR DESCRIPTION
**Pull request by BLasan**
Description
Completed View All Floating-Rates component using mat-table.

Related issues and discussion
#671 

Screenshots, if any
Screenshot from 2020-03-21 13-52-57

Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

File: src/app/products/floating-rates/floating-rates.component.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  /**
+   * Initializes the data source, paginator and sorter for floating rates table.
+   */
+  setFloatingRates() {
+    this.dataSource = new MatTableDataSource(this.floatingRatesData);
+    this.dataSource.paginator = this.paginator;
+    this.dataSource.sort = this.sort;
+  }
+
+
```
Comment (karantakalkar) [inline src/app/products/floating-rates/floating-rates.component.ts]:
```suggestion

```

File: src/app/products/floating-rates/floating-rates.resolver.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import { ProductsService } from '../products.service';
+
+/**
+ * Loan products data resolver.
+ */
+@Injectable()
+export class FloatingRatesResolver implements Resolve<Object> {
+
+  /**
+   *
```
Comment (karantakalkar) [inline src/app/products/floating-rates/floating-rates.resolver.ts]:
```suggestion
```

File: src/app/products/floating-rates/floating-rates.component.ts [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { MatPaginator, MatSort, MatTableDataSource } from '@angular/material';
+import { ActivatedRoute } from '@angular/router';
+
```
Comment (karantakalkar) [inline src/app/products/floating-rates/floating-rates.component.ts]: add all comments as mentioned in #674

File: src/app/products/floating-rates/floating-rates.component.scss [hunk 4]
```diff
@@ -0,0 +1,8 @@
+table {
+    width: 100%;
+  
+    .select-row:hover {
+      cursor: pointer;
+    }
+  }
+
```
Comment (karantakalkar) [inline src/app/products/floating-rates/floating-rates.component.scss]:
```suggestion
  
```

**Comment by Jov03** [comment]
@BLasan this issue was already logged issue #194 and PR for the same was already present.

**Pull request closed without merging**